ui/lib/core/addon/components/replication-page.js

```diff
@@ -29,22 +30,45 @@ export default Component.extend({
     }
     this.set('reindexingDetails', resp);
   }),
+  isSummaryDashboard: computed('model.dr.{mode}', 'model.performance.{mode}', function() {
+    const router = this.router;
+    const currentRoute = router.get('currentRouteName');
+
+    // we only show the summary dashboard in the replication index route
+    if (currentRoute === 'vault.cluster.replication.index') {
+      const drMode = this.model.dr.mode;
+      const performanceMode = this.model.performance.mode;
+      return drMode && performanceMode === 'primary';
```

Comment:
Do you want to check if the dr mode is also primary? In that case this line should be `return drMode === 'primary' && performanceMode === 'primary'`

Reply:
Nice catch!  This one was a slippery one to catch because we only compute this property when the summary dashboard shows, which only happens when both are a primary so by default it was correctly returning true, but not because drMode was a primary, but because peroformanceMode always was.  I'll tag @noelledaley  as I believe she wrote this.